### DISCUSSION TITLE
Proposal for Change in DOCS.md (problem with Enroll public key in your vehicle)

### DISCUSSION THREAD

**Discussion by tomarentz** [llamafilm/tesla-http-proxy-addon#62] [Ideas]
Hi, Thank you @llamafilm for this great add-on :-) 
I have just after 3 days of learning and failing managed to get this working. My last struggle was enrolling in the Tesla app. Following the stepps, I got this message in the Tesla app:
![image](https://github.com/llamafilm/tesla-http-proxy-addon/assets/142660339/74c604b3-a186-4837-a99f-763f2c7a73d8)
The problem was (as the message says) that my public key was not authorized in my account. It did not show up here: https://accounts.tesla.com/en_US/account-settings/security?tab=tpty-apps
In my case this authorization happened when I used the [Tesla Auth app](https://apps.apple.com/us/app/auth-app-for-tesla/id1552058613) to obtain a refresh token. After that everything works fine.
I expected this to happen with the "Regenerate Tesla authentication" toggle in the config. If not, then I suggest that you put the line with Tesla Auth app before the section with "Enroll public key in your vehicle" in the DOCS.md file.

**Comment by bvlaicu** [comment]
I agree. I had the same problem. Thanks for bringing this up, @tomarentz. It helped unblock me.

**Comment by llamafilm** [comment]
Thanks for the feedback, I updated the docs.
